src/Services/UserManagementService/UsersManagementService.DAL/Entites/PagedUsersResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UsersManagementService.DAL.Entites;
+
+public class PagedUsersResponse
```

Comment:
naming "response" is more suitable for Presentation level. For DAL something like "Projection" is better